tests/integ/sagemaker/serve/test_serve_transformers.py

```diff
@@ -106,7 +106,7 @@ def test_pytorch_transformers_sagemaker_endpoint(
     with timeout(minutes=SERVE_SAGEMAKER_ENDPOINT_TIMEOUT):
         try:
             logger.info("Deploying and predicting in SAGEMAKER_ENDPOINT mode...")
-            predictor = model.deploy(instance_type=gpu_instance_type, initial_instance_count=1)
```

Comment:
same comment

Reply:
Thanks, No that is a random instance, i can change it add variability